portfolio/src/main/webapp/homepage.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Google LLC
```

Comment:
Same as above. We don't need this license.

Reply:
Fixed.